Azure/azure-cli
# [Core] Built-in-alias

**Description**
Support alias in command level and argument level

**Testing Guide**
`az tsi env -h`
`az tsi env create --type longterm`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
